src/test/scala/org/constellation/storage/CheckpointServiceTest.scala

```diff
@@ -120,6 +120,33 @@ class CheckpointServiceTest
         .unsafeRunSync()
       dao.checkpointService.contains(cb3.baseHash).unsafeRunSync() shouldBe true
     }
+
+    "should accept cb resolving parents created with dummy transactions" in {
```

Comment:
Wouldn't be better to add parameter to `makeBlock` function (with default random transaction) instead of creating separate function ?